.eslintrc

```diff
@@ -39,13 +39,51 @@
     ],
     "@typescript-eslint/camelcase": "off",
     "@typescript-eslint/consistent-indexed-object-style": "error",
+    "@typescript-eslint/consistent-type-assertions": "error",
     "@typescript-eslint/consistent-type-definitions": ["error", "interface"],
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/member-ordering": [
```

Comment:
I use this in my personal project (and import ordering).  95% of the time, it does what I want.  5% of the time it doesn't.  Personally, I'm cool with this.  That being said, if you don't include the autofix, it's incredibly annoying...

Reply:
Yeah the only "undesirable" change in this PR was moving `__type` from the first property of `ThemeType` to the last, which I thought was okay.